conda_libmamba_solver/solver.py

```diff
@@ -551,24 +510,29 @@ def _specs_to_tasks_add(self, in_state: SolverInputState, out_state: SolverOutpu
                                 lock = False
                                 break
                     if lock:
-                        tasks[LOCK].append(installed_spec_str)
+                        tasks[Request.Freeze].append(installed_spec)
+                    # enabling this else branch makes
+                    # conda/conda's tests/core/test_solve.py::test_freeze_deps_1[libmamba] fail
+                    # else:
+                    #     tasks[Request.Keep].append(name)
```

Comment:
What should we do about it?

Reply:
I think we should leave it as is. Simpler that way.